### PR TITLE
added Battery live status

### PR DESCRIPTION
**Pull request by coolguy08**
Hii , Updated battery status using Battery status API.

### CODE REVIEW

**Comment by andrewstech** [comment]
Ignore the failed windows check.

**Comment by coolguy08** [comment]
hii , @andrewstech can you tell me why my changes are not there in the preview deploy. https://124.pr.blueedge.me

**Comment by Lukas-Batema** [comment]
Is it just supposed to be an image??? 
image

**Comment by Lukas-Batema** [comment]
> hii , @andrewstech can you tell me why my changes are not there in the preview deploy. https://124.pr.blueedge.me

Maybe, check your scss/css padding or related things, since it may not be in the display...

**Comment by blueedgetechno** [comment]
Does this take in account of battery charging or not? If you implement, maybe try doing it fully. You can use dynamic svg for this rather that different icons for different range.

**Comment by coolguy08** [comment]
> Does this take in account of battery charging or not? If you implement, maybe try doing it fully. You can use dynamic svg for this rather that different icons for different range.

yes it has the charging case.

**Comment by andrewstech** [comment]
> hii , @andrewstech can you tell me why my changes are not there in the preview deploy. https://124.pr.blueedge.me

Not sure have you tried a local build? ``` npm run build ```

**Comment by blueedgetechno** [comment]
I mean, it shows charging even My device is not charging. What do you mean by charging case?

**Comment by Lukas-Batema** [comment]
> I mean, it shows charging even My device is not charging. What do you mean by charging case?

And it should have a 0 percent on Firefox/non-supported browsers, and it shows charging, even when it isn't and full(ish) on my Firefox for iOS... sooo... yeah.

**Comment by coolguy08** [comment]
> I mean, it shows charging even My device is not charging. What do you mean by charging case?

Because the changes were not deployed in the preview. https://124.pr.blueedge.me/.
I don't know why?

**Comment by coolguy08** [comment]
> > hii , @andrewstech can you tell me why my changes are not there in the preview deploy. https://124.pr.blueedge.me
> 
> Not sure have you tried a local build? `npm run build`

Yes , it is working fine. But the code that I have changed are not there in the preview deploy . Not even the images that I have uploaded.

**Comment by coolguy08** [comment]
> > I mean, it shows charging even My device is not charging. What do you mean by charging case?
> 
> And it should have a 0 percent on Firefox/non-supported browsers, and it shows charging, even when it isn't and full(ish) on my Firefox for iOS... sooo... yeah.

Because I have used charging img as default for not supported browsers.
Showing 0% battery to non-supported browser would be irrelevant.

**Comment by Lukas-Batema** [comment]
> > > I mean, it shows charging even My device is not charging. What do you mean by charging case?
> 
> > 
> 
> > And it should have a 0 percent on Firefox/non-supported browsers, and it shows charging, even when it isn't and full(ish) on my Firefox for iOS... sooo... yeah.
> 
> 
> 
> Because I have used charging img as default for not supported browsers.
> 
> Showing 0% battery to non-supported browser would be irrelevant.

Well, if it's not charging, it isn't relevant, and the demo site in the issue by you, shows 0% on a non-supported browser...

**Comment by Lukas-Batema** [comment]
> Ignore the failed windows check.

\- @andrewstech 

---
> hii , @andrewstech can you tell me why my changes are not there in the preview deploy. [124.pr.blueedge.me](https://124.pr.blueedge.me)

\- @coolguy08
---
That may be the reason changes are not showing up, the failed window check. Just saying...

**Review by Lukas-Batema — CHANGES_REQUESTED** [review]
Can you change all the PNGs to dynamic SVGs, as @blueedgetechno said? Also, change the references to those images, so it will update. Yes, I know I am not the one who should be reviewing, but I found a lack of professionalism within this repository, in the pr review category, so I decided to start reviewing PRs as much as possible ;)

**Comment by andrewstech** [comment]
> > Ignore the failed windows check.
> 
> - @andrewstech
> 
> > hii , @andrewstech can you tell me why my changes are not there in the preview deploy. [124.pr.blueedge.me](https://124.pr.blueedge.me)
> 
> - @coolguy08
> 
> That may be the reason changes are not showing up, the failed window check. Just saying...

Nope. This is separate from the Deploy preview system. This check sees if it can build an exe via electron.

**Comment by andrewstech** [comment]
Closed then reopened to run CI

**Comment by andrewstech** [comment]
Hey @coolguy08 & @Lukas-Batema The preview has been fixed. It was due to a bug in my code where it was deploying the master of this repo not the PR.

**Comment by Lukas-Batema** [comment]
> Hey @coolguy08 & @Lukas-Batema The preview has been fixed. It was due to a bug in my code where it was deploying the master of this repo not the PR. 

Ah, ok, thank you!

**Review by Tronic247 — APPROVED** [review]
Cool

**Comment by andrewstech** [comment]
The test branch is where new features get added before release. You can view this online at https://dev.blueedge.me/

**Comment by andrewstech** [comment]
Not sure if this is just showing for me but it seems to overlapping.
image

**Comment by blueedgetechno** [comment]
same for me, 
image

**Comment by Lukas-Batema** [comment]
I get this issue in chrome on iOS 15 
https://user-images.githubusercontent.com/55695493/134359010-c540bc75-2fb5-42b3-b872-4a25899af82b.MOV

**Comment by Ccode-lang** [comment]
This is added to only show on phones. It is perfectly normal and the dev said they will fix it soon.

**Comment by Lukas-Batema** [comment]
> This is added to only show on phones. It is perfectly normal and the dev said they will fix it soon.

I know that message is meant for that, the screen sizing when switched to landscape mode, and **NOT** showing the message is the problem, which is shown in the video.

**Comment by coolguy08** [comment]
Hii @andrewstech  & @blueedgetechno fixed the icon overlap issue.

**Comment by andrewstech** [comment]
> Hii @andrewstech & @blueedgetechno fixed the icon overlap issue.

Great. :)

**Review by andrewstech — APPROVED** [review]
👍

**Review by Lukas-Batema — APPROVED** [review]
It looks good now :)

**Comment by Lukas-Batema** [comment]
> **This PR is awaiting approval from @blueedgetechno**.
> 
> _THIS BOT IS POWERED BY GITHUB ACTIONS & A DEBIAN VM_

Lmao, it has taken this long for @blueedgetechno to even review this PR?

File: public/index.html [hunk 1]
```diff
@@ -14,6 +14,7 @@
   <title>Windows 11 in React</title>
   <link rel="stylesheet" href="dycalendar.css">
   <link rel="stylesheet" href="https://static2.sharepointonline.com/files/fabric/office-ui-fabric-core/11.0.0/css/fabric.min.css"/>
+  <link href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.6.3/css/font-awesome.min.css" rel="stylesheet" type="text/css" />
```
Comment (yashash1511) [inline public/index.html:17]: was this pr even reviewed?

Comment (Lukas-Batema) [reply public/index.html:17]: LMAO, I didn't notice that...

Comment (Lukas-Batema) [reply public/index.html:17]: The CSS sheet is pointless, isn't it? Probably is slowing it down...

**Pull request merged**